cmd/format-release-notes/formatter_test.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestFormatReleaseNotesMarkdown(t *testing.T) {
+	type testCase struct {
+		name     string
+		markdown []byte
+		expected []byte
+	}
+	cases := []testCase{
+		{
+			name: "transforms relative links",
+			markdown: []byte(`### PMM 2.42.0
+
+Welcome to PMM [v2.42](../index.md)
+
+-![!image](../_images/Max_Connection_Limit.png)`),
+			expected: []byte(`### PMM 2.42.0
+
+Welcome to PMM [v2.42](https://github.com/percona/pmm-doc/tree/main/docs/index.md)
```

Comment:
should it lead to github instead of https://docs.percona.com/percona-monitoring-and-management/3/index.html?
BTW we already migrated from percona/pmm-doc to percona/pmm

Reply:
yeah, the reason here was that different relative markdown links can link to different links (not just release notes for instance), and it's hard to define the proper link

Reply:
@idoqo can you provide a few example in existing release notes, please?
@catalinaadam can you help with this, please?

Reply:
Yeah here are a few:
* https://github.com/percona/pmm/blob/aeb37dfd26adec1befd23f877fa7148b189c1264/documentation/docs/release-notes/3.0.0.md?plain=1#L54
* https://github.com/percona/pmm/blob/aeb37dfd26adec1befd23f877fa7148b189c1264/documentation/docs/release-notes/3.0.0.md?plain=1#L6
* https://github.com/percona/pmm-doc/blob/bf87735acab5d714a8d1266be47d6680e6637893/docs/release-notes/2.16.0.md?plain=1#L36
* https://github.com/percona/pmm-doc/blob/159ad201a7c66bc92964e017cdabc4fd0fd90451/docs/release-notes/2.12.0.md?plain=1#L25

Reply:
all of them in docs.percona.com, there are no plans to use /2/ links for PMM 3 release notes.

Reply:
I've updated those to use the docs.percona.com links now